blocks/columns/columns.css

```diff
@@ -41,8 +42,8 @@
 }
 
 .columns.gray {
-  > div > div {
-    background: var(--divider-gray);
+  > div > div:has(p), > div > div:has(a), > div > div:has(h2) {
```

Comment:
I don't really understand why the column can only contain these things when gray? What if there's just a picture or button in a column?